libimage/runtime_test.go

```diff
@@ -56,6 +58,13 @@ func testNewRuntime(t *testing.T, options ...testNewRuntimeOptions) *Runtime {
 	return runtime
 }
 
+func testRuntimePullImage(t *testing.T, r *Runtime, ctx context.Context, imageName string) {
+	pullOptions := &PullOptions{}
+	pullOptions.Writer = os.Stdout
+	_, err := r.Pull(ctx, imageName, config.PullPolicyAlways, pullOptions)
```

Comment:
Does this need to be `PullPolicyAlways`? Many other tests use `PullPolicyMissing`; that might be faster. (Or, in the tests where we are setting up a new runtime, both do the same thing but Missing doesn’t hurt.)